contracts/0.8.9/DepositSecurityModule.sol

```diff
@@ -200,6 +201,7 @@ contract DepositSecurityModule {
     }
 
     function _setMinDepositBlockDistance(uint256 newValue) internal {
+        require(newValue > 0, "invalid value");
```

Comment:
Maybe change to something readable? Like "minDepositBlockDistance should be greater than zero"?
WDYT?